Src/WitsmlExplorer.Api/Workers/Delete/DeleteObjectsWorker.cs

```diff
@@ -74,5 +83,6 @@ public async Task<(WorkerResult, RefreshAction)> DeleteObjectsOnWellbore(IWitsml
                 ? (new WorkerResult(witsmlClient.GetServerHostname(), true, successString), refreshAction)
                 : (new WorkerResult(witsmlClient.GetServerHostname(), false, $"{successString} Failed to delete some {queries.First().GetType().Name}s", errorReason, null), successUids.Count > 0 ? refreshAction : null);
```

Comment:
null is already the default value

Reply:
Is there any better way to write this line?